src/parser.c

```diff
@@ -221,6 +222,12 @@ void drop_quotes(char **str)
     }
 }
 
+void skip_next_spaces(char **str)
```

Comment:
How about simple `\brief` for this function?

Reply:
Not now yet.